src/polyswarmd/config/status.py

```diff
@@ -0,0 +1,27 @@
+from typing import List, Dict
+
+from polyswarmd.config.service import Service
+
+
+class Status:
+    community: str
+    services: List[Service]
+
+    def __init__(self, community):
+        self.community = community
+        self.services = []
+
+    def get_status(self):
+        status = {'community': self.community}
+        status.update(self.test_services())
```

Comment:
Because `Service.get_service_state` returns a `Dict[str, ...]`, so I think this is actually merging a (possibly) nested dict, e.g `Dict[str, Union[str, bool, Dict]]`.

Taking a quick look at what `/status` returns, I believe is correct, but make sure you check polyswarm/polyswarmd#271's typing changes to see if they make sense.